_posts/2023-01-12-January-Update-Finishing-my-Igalia-CE.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "January Update: Finishing my Igalia CE"
+date: 2023-01-12
+author: Maíra Canal
+layout: post
+permalink: /january-update-finishing-my-igalia-ce/
+categories: Genel
+tags: [igalia, graphics]
+---
+
+2022 really passed by fast and I'm again finishing another mentorship project.
```

Comment:
I'd avoid this "I'm again finishing another...". I'd rewrite this idea as accomplishments/awards: "and after I completed the GSoC 2022, I'm now completing another milestone: my project in the Igalia Coding Experience"